### ISSUE TITLE
One more bug with Gunner

### ISSUE THREAD

**Issue by Sandrem** [Sandrem/FlyCasual#495]
>> A bug I'm running into while playing Whisper with Veteran Instincts, Fire Control System, Gunner, and Advanced Cloaking device. At the end of the attack phase when you Attack (Miss) > Fire Control System > Gunner Attack > Whisper Focus > Advanced Cloak I get a red blob of text similar to what FM posted above. I have not found a way to hide the console once it comes up.

**Comment by Vazkman** [comment]
Probably it's related to any extra attack. It happened to my BroBots as well.

![second attack bug](https://user-images.githubusercontent.com/1640572/35197313-7bc2c1c6-fec4-11e7-8eb6-d8b89d3666dd.PNG)

**Comment by Sandrem** [comment]
Yes. Seems that TLT, IG-88B, Gunner, TIE/D cause bugs. In 0.4.0 I added ability of extra attack (Corran Horn, IG-88B) and broke this part of code. I plan to fix it in ~ mid-week.
